src/utilities.ts

```diff
@@ -394,85 +394,112 @@ function exeFilename(basename: string): string {
   else return basename;
 }
 
-function findVcastTools() {
-  // This function will check if vPython is on the path or if the
-  // VectorCAST Installation directory option is set
-  // In either of these "positive" cases it will setup the
-  // vPythonCommandToUse and clicastCommandtoUse global variables
-  // otherwise it will display and error.
+function findVcastTools():boolean {
+
+  // This function will set global paths to vpython, clicast and vcastqt
+  // by sequentially looking for vpython in the directory set via the 
+  // 1. extension option: "vcastInstallationLocation"
+  // 2. VECTORCAST_DIR
+  // 3. system PATH variable
+
+
+  // return valuo
+  let foundAllvcastTools = false;
+
+  // value of the extension option
+  const settings = vscode.workspace.getConfiguration("vectorcastTestExplorer");
+  const installationOptionString = settings.get("vcastInstallationLocation", "");
+
+  // value of VECTORCAST_DIR
+  const  vectorcastDirString = process.env ["VECTORCAST_DIR"];
 
-  // check the value of the option, I know this is extra work if
-  // vcast is on the path but this is done once and it make the code cleaner
-  let settings = vscode.workspace.getConfiguration("vectorcastTestExplorer");
-  const installationLocation = settings.get("vcastInstallationLocation", "");
 
-  let vcastToolsOK: boolean = true;
+  // possible installation location
+  let vcastInstallationPath:string|undefined = undefined;
 
-  if (vpythonOnPath) {
-    vPythonCommandToUse = vPythonName;
-    vectorMessage(`   found '${vPythonName}' on the system PATH ...`);
-  } else if (installationLocation.length > 0) {
+  // priority 1 is the option value, since this lets the user over-ride PATH or VECTORCAST_DIR
+  if (installationOptionString.length > 0) {
```

Comment:
For readability, we could refactor this to something like:
```
if (installationOptionString.length > 0) 
  setVcastInstallPathUsingOption(installationOptionString);
else if (vectorcastDirString)
  setVcastInstallPathUsingVcastDir(vectorcastDirString);
else 
  setVcastInstallPathUsingSystemPath(vpythonOnPath);

foundAllVcastTools = False;
if (vcastInstallationPath)
  foundAllVcastTools = findVcastExecutables(vcastInstallationPath)

return foundAllVcastTools
```

Reply:
Ya, I considered a few other approaches, including a functions for each, but since there are only 3 choices I like it better in one place where I can see what we do for each one.